src/components/presentational/SingleSurveyTask/SingleSurveyTask.tsx

```diff
@@ -17,75 +16,98 @@ import { ButtonVariant } from '../Button/Button';
 import checkMark from '../../../assets/greenCheck.svg';
 import Action from '../Action';
 import LoadingIndicator from '../LoadingIndicator';
+import { getLocale } from '../../../helpers/date-helpers';
+import { noop } from '../../../helpers/functions';
```

Comment:
I'm not clear on what noop is doing? Is there a different way to accomplish what it is doing that does not involve a blank file + import?

Reply:
You can see the usage down here (`Button` requires a click listener, but I don't want it to do anything when clicked):  https://github.com/CareEvolution/MyDataHelpsUI/pull/177/files/feebf11de5146de56601851c90c9b37568d203ee#diff-dcf4df28a6983ef66aeb389be69aee68e9b916c3823798af4cb6b827d10900a4R74

I just thought it looked cleaner in the code to have a specific "no-op" function available rather than using `() => {}` all over the place.  The use of `noop` looks more intentional to me, and less like a click listener that someone forgot to finish implementing.